### PR TITLE
Added networking view, get route for conversations

### PR DESCRIPTION
**Pull request by awilli70**
Current View:
Screen Shot 2020-07-01 at 12 29 17 AM

### CODE REVIEW

File: server/models/conversationSchema.js [hunk 1]
```diff
@@ -16,7 +16,9 @@ const conversationSchema = new Schema(
         alumni: [{type: Schema.Types.ObjectId, ref: 'Alumni'}],
         messages: [messageSchema],
         seen: [{type: Boolean}],
-        dateCreated: {type: Date, default: moment()}, 
+        dateCreated: {type: Date, default: moment()},
+        //RETURN FIELD NOT KEPT IN DB
+        timeFromMessage: {type: String}
```
Comment (awilli70) [inline server/models/conversationSchema.js:21]: comment is pretty self explanatory, but this field is a workaround for returning fields that don't need to be stored on the database

I looked into mongoose's virtuals, and while they have the right functionality they aren't passed in a response unless I go through and convert every document to JSON before sending the response

File: client/src/components/AlumniNetworking.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { Card, Image, Grid, Label, icon } from 'semantic-ui-react'
```
Comment (reshadbinharun) [inline client/src/components/AlumniNetworking.js:2]: `icon` import may not be needed if used as a prop on `Label`

File: client/src/components/AlumniNetworking.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        let conversations = await this.getConversations()
+        this.setState({
+            timeOffsetHours: timeOffsetHours,
+            conversations: conversations.conversations
+        })
+        this.createDisplay(conversations.conversations)
+    }
+
+    getConversations() {
+        return makeCall({}, '/conversations/get/' + this.props.userDetails._id, 'get')
```
Comment (reshadbinharun) [inline client/src/components/AlumniNetworking.js:26]: `/get/` part of path may be redundant

File: server/routes/conversations.js [hunk 4]
```diff
@@ -41,11 +42,29 @@ router.post('/add/', passport.authenticate('jwt', {session: false}), async (req,
...
     } catch (e) {
-        console.log('request/addConversation Error: ' + e)
+        console.log('conversations/add Error: ' + e)
         res.status(500).send({
             message: 'Failed creating conversation: ' + e
         });
     }
 });
 
+router.get('/get/:id', passport.authenticate('jwt', {session: false}), async (req, res) => {
```
Comment (reshadbinharun) [inline server/routes/conversations.js:52]: `/get` may be redundant here

**Pull request merged**